core/app/models/spree/order/store_credit.rb

```diff
@@ -0,0 +1,87 @@
+module Spree
+  class Order < Spree::Base
+    module StoreCredit
+      def add_store_credit_payments
```

Comment:
Assignment Branch Condition size for add_store_credit_payments is too high. [21.21/15]